### PR TITLE
Add brackets to the III in Iron(III) Chloride

### PR DESCRIPTION
**Pull request by tracer4b**

### CODE REVIEW

**Review by ALongStringOfNumbers — APPROVED** [review]

File: src/main/resources/assets/gregtech/lang/en_us.lang [hunk 1]
```diff
@@ -1575,7 +1575,7 @@ material.hypochlorous_acid=Hypochlorous Acid
 material.ammonia=Ammonia
 material.hydrofluoric_acid=Hydrofluoric Acid
 material.nitric_oxide=Nitric Oxide
-material.iron_iii_chloride=Iron III Chloride
+material.iron_iii_chloride=Iron(III) Chloride
```
Comment (TechLord22) [inline src/main/resources/assets/gregtech/lang/en_us.lang:1578]:
```suggestion
material.iron_iii_chloride=Iron (III) Chloride
```

**Review by TechLord22 — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/gregtech/lang/en_us.lang (hunk 1 above)
Comment (LAGIdiot) [reply src/main/resources/assets/gregtech/lang/en_us.lang:1578]: I am not sure about the space between Iron and (III) as from quick search most places are using it without it.

**Comment by tracer4b** [comment]
I plan to do one big material names/formulas PR soon (after ore rework) and include this with it. It doesn’t need to go in right away but it can if you want

**Comment by LAGIdiot** [comment]
Honestly I don't care about this kind of content (am not science guy, this is just not my thing). What I care about is number of open PRs, which needs to be checked and handled.

**Comment by tracer4b** [comment]
Then I will close this for now

**Pull request closed without merging**